keyboards/xbows/woody/info.json

```diff
@@ -0,0 +1,5 @@
+["","","","","","","","","","","","","",{w:2},"",""],

```

Comment:
This file is in the wrong format.

Please use https://qmk.fm/converter/ to turn it into a proper info.json.